src/styles/tag.less

```diff
@@ -0,0 +1,29 @@
+@tag-text-vertical-padding: 3px;
+
+.Tag {
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+  padding: @tag-text-vertical-padding 16px;
+  border-radius: 22px;
+  background-color: @grey-3;
+  font-size: 12px;
+  margin: 0 5px;
+  border: none;
+
+  &:first-of-type {
+    margin-left: 0;
+  }
+  &:last-of-type {
+    margin-right: 0;
+  }
+
+  &.Tag--selected {
+    color: @edu-white;
+    background-color: @edu-primary-color;
+
+    & > .ant-tag-close-icon {
```

Comment:
Sadly there is no other way to overwrite ant design's style here, even when we use our own custom icon.